service/service.go

```diff
@@ -11,16 +11,18 @@ import (
 )
 
 type Service struct {
-	repo                           repository
+	repo                           Repository
 	privateKey                     *rsa.PrivateKey
-	currentUserInformationProvider currentUserInformationProvider
+	currentUserInformationProvider СurrentUserInformationProvider
 }
 
-type currentUserInformationProvider interface {
+//go:generate mockgen -destination=provider_mock_test.go -package=service . CurrentUserInformationProvider
+type СurrentUserInformationProvider interface {
 	GetCurrentUserID(ctx context.Context) string
 }
 
-type repository interface {
+//go:generate mockgen -destination=repo_mock_test.go -package=service . Repository
+type Repository interface {
```

Comment:
please don't change `repository` to `Repository`, and the same to all interfaces